modules/rosa-checking-account-version-info-cli.adoc

```diff
@@ -1,13 +1,7 @@
+[id="rosa-checking-account-version-information_{context}"]
+= Checking account and version information with the rosa CLI
 
-// Module included in the following assemblies:
```

Comment:
This is still required metadata: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata